components/TextArea/TextArea.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { INPUT_STYLE, Label, FieldGroup, ErrorMessage } from '../shared';
+import uniqId from '../shared/uniqId';
+
+const {
+  default: DEFAULT_INPUT_STYLE,
+  LABEL_STYLE,
+  HELPER_TEXT_STYLE,
+  REQUIRED_MARK_STYLE,
+  ERROR_MESSAGE_STYLE,
+  AUTO_FILL_STYLE,
+} = INPUT_STYLE;
+
+const TextAreaTag = styled.textarea`
+  ${DEFAULT_INPUT_STYLE};
+
+  display: block;
```

Comment:
I think the default display value of textarea element is already `block`.

Reply:
Unfortunately, no :cry: 

![Screenshot from 2019-03-22 08-57-06](https://user-images.githubusercontent.com/755101/54821317-8dcc1d80-4c80-11e9-8976-d8347618056c.png)

Reply:
:disappointed: